lib/renderer/security-warnings.ts

```diff
@@ -81,7 +81,7 @@ const isUnsafeEvalEnabled: () => Promise<boolean> = function () {
   // Call _executeJavaScript to bypass the world-safe deprecation warning
   return (webFrame as any)._executeJavaScript(`(${(() => {
     try {
-      new Function(''); // eslint-disable-line no-new,no-new-func
+      eval((window as any).trustedTypes.emptyScript); // eslint-disable-line no-eval
```

Comment:
Nit: To be future-proof (e.g. in case this code ever runs on a browser engine without Trusted Types) you may want to check if `window.trustedTypes` exists, and simply use an empty string instead, e.g. like so:

```
eval((window as any).trustedTypes?.emptyScript ?? '')
```

This relies on the fact that `emptyScript`, while wrapping over an empty string, is truthy.

Reply:
Trusted types are supported since Chromium 83, which is Electron 9. Therefore there is no need for a check